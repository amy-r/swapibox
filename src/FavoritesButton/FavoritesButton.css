.FavoritesButton {
  background-color: #458BCA;
  border: none;
  box-shadow: none;
  height: 60px;
  border-radius: 3px;
  color: white;
  letter-spacing: 3px;
}