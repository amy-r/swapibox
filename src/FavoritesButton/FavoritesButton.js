import React from 'react';
import propTypes from 'prop-types';
import './FavoritesButton.css'

const FavoritesButton = (props) => {
    return (
      <button onClick = {props.onClick} className='FavoritesButton'> FAVORITES </button>
    )
}

export default FavoritesButton;