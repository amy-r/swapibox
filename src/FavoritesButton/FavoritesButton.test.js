/* eslint-disable */ 

import React from 'react';
import ReactDOM from 'react-dom';
import FavoritesButton from './FavoritesButton';
import { shallow, mount } from 'enzyme';
import renderer from 'react-test-renderer';

describe( 'Favorites Button', () => {

  it('exists', () => {
  })

  it('renders without crashing', () => {
    const renderedComponent = shallow(<FavoritesButton />, {disableLifecycleMethods: true})
    expect(renderedComponent).toMatchSnapshot();
  })

});