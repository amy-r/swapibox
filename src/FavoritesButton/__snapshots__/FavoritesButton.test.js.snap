// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorites Button renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <FavoritesButton />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": " FAVORITES ",
      "className": "FavoritesButton",
      "onClick": undefined,
    },
    "ref": null,
    "rendered": " FAVORITES ",
    "type": "button",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": " FAVORITES ",
        "className": "FavoritesButton",
        "onClick": undefined,
      },
      "ref": null,
      "rendered": " FAVORITES ",
      "type": "button",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
